docs/modules/ROOT/pages/advanced-config.adoc

```diff
@@ -71,6 +71,14 @@ spec:
 
 ----
 
+[TIP]
+.Passing environment variables
+====
+If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. Example:
+
+`appuio.ch/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'" --overwrite`
```

Comment:
```suggestion
`kubectl -n wordpress-demo annotate pods ${YOUR_POD_NAME} "appuio.ch/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'" --overwrite`
```
without the kubectl it would be very confusing and have an open `"` and an unexplained `--overwrite`

Reply:
```suggestion
`"appuio.ch/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'"`
```
As an alternative document this.